packages/mf2-messageformat/src/functions/number.ts

```diff
@@ -61,23 +61,18 @@ export function number(
     Intl.PluralRulesOptions & { select?: 'exact' | 'cardinal' | 'ordinal' } = {
     localeMatcher
   };
-  switch (typeof input) {
-    case 'bigint':
-    case 'number':
-      value = input;
```

Comment:
This is left out in the change, and is necessary for numeric inputs.

Reply:
Oh I see - I didn't realize value and input were separate variables. They are not separate in the spec. Maybe I could align them here?

Reply:
Aligning them would be fine, if you can get TS to be satisfied with that...

Reply:
Actually, it might've been not wanting to change an argument value that drove this design, and so changing line 59 to something like
```ts
let value = input;
```
would probably be fine.

Reply:
OK will do.